standard_contracts/src/foreign_contracts.rs

```diff
@@ -176,11 +176,29 @@ pub mod core {
 
         pub mod implement_usize {
             pub fn max_value() -> usize {
-                4294967295
+                if cfg!(any(
```

Comment:
Perhaps just return the usize::MAX constant?

Reply:
This might not currently be possible. See issue #241